packages/cosmic-swingset/provisioning-server/src/ag_pserver/main.py

```diff
@@ -205,39 +204,16 @@ def enablePubkey(reactor, opts, config, nickname, pubkey):
     def ret(server_message):
         return [mobj, server_message, config]
 
-    # FIXME: Make more resilient to DOS attacks, or attempts
-    # to drain all our uags.
     args = [
-        'tx', 'send', '--keyring-backend=test', config['bootstrapAddress'], pubkey, opts['initial-token'],
-        '--yes', '--broadcast-mode', 'block', # Don't return until committed.
+        'tx', 'swingset', 'provision-one', '--keyring-backend=test', nickname, pubkey,
+        '--from=ag-solo', '--yes', '--broadcast-mode=block', # Don't return until committed.
+        '--gas=auto', '--gas-adjustment=1.4',
     ]
     code, output = yield agCosmosHelper(reactor, opts, config, args, 10)
     if code != 0:
         return ret({"ok": False, "error": 'transfer returned ' + str(code)})
 
-    controller_url = opts["controller"]
-    print('contacting ' + controller_url)
-    m = json.dumps(mobj)
-
-    # this HTTP request goes to the controller machine, where it should
-    # be routed to vat-provisioning.js and the pleaseProvision() method.
-    try:
-        resp = yield treq.post(controller_url, m.encode('utf-8'), reactor=reactor,
-                                headers={
-                                    b'Content-Type': [b'application/json'],
-                                    b'Origin': [b'http://127.0.0.1'],
-                                })
-        if resp.code < 200 or resp.code >= 300:
-            raise Exception('invalid response code ' + str(resp.code))
-        rawResp = yield treq.json_content(resp)
-    except Exception as e:
-        print('controller error', e)
-        return ret({"ok": False, "error": str(e)})
-    if not rawResp.get("ok"):
-        print("provisioning server error", rawResp)
-        return ret({"ok": False, "error": rawResp.get('rej')})
-    r = rawResp['res']
-    ingressIndex = r["ingressIndex"]
+    ingressIndex = 1
```

Comment:
style: `ingress_index`

I believe Python’s a prevailing snake_case and BactrianCase sort of language, but it looks like Twisted ascribes to a different guide.

Reply:
Twisted is very `camelCase` for methods and `PascalCase` for classes, and somewhat `camelCase` for variables. Personally I'm more `snake_case` than `camelCase`, which is why it shows up in magic-wormhole (I'm `kebab-case` for filenames and command/subcommand names). The portions of this code that I wrote did a bit of style-shifting depending upon their proximity to the two different libraries.

FYI I was delighted to learn names for these styles (from the Rust [serde](https://serde.rs/container-attrs.html)) library, especially putting a name to kebab-case. My favorite is SCREAMING_SNAKE_CASE.